e2e/spec/missing-slot.spec.e2e.ts

```diff
@@ -37,6 +37,7 @@ Scenario('(CSR) should correctly open page where app requires missing slot in te
 
     const consoleMessages: ConsoleMessage[] = await I.grabBrowserLogs();
     const messagesText = consoleMessages.map(v => v.text());
+    console.log("🚀 ~ file: missing-slot.spec.e2e.ts:40 ~ Scenario ~ messagesText:", messagesText)
```

Comment:
Leftover or we do need this comment in prod ?

Reply:
Debugging e2e test, already removed.